---
title: "Mapping the functional genomic landscape de novo with divergent transcription analysis of PRO-seq data"
author: "Domenica D'Elia"
authors:
  - name: Domenica D'Elia 
    affiliations:
      - ref: u1
  - name: Cristina Pizzulli 
    affiliations:
      - ref: u1
  - name: Veronica Butaro 
    affiliations:
      - ref: u2
  - name: Daniele Rosa 
    affiliations:
      - ref: u2
  - name: Antonio Pellicani
    affiliations:
      - ref: u2
  - name: Gianvito Pio 
    affiliations:
      - ref: u2
      - ref: u3
  - name: Michelangelo Ceci
    affiliations:
      - ref: u3
      - ref: u3
      - ref: u4
affiliations:
  - id: u1
    name: Institute for Biomedical Technologies, National Research Council, Bari, Italy
  - id: u2
    name: Dept. of Computer Science, University of Bari, Bari, Italy
  - id: u3
    name: Big Data Lab, National Interuniversity Consortium for Informatics (CINI), Rome, Italy
  - id: u4
    name: Jožef Stefan Institute, Ljubljana, Slovenia
date: 2024-11-04T14:45:25
categories:
  - poster session
  - monday
---

## Abstract: 

The rising prevalence of chronic diseases poses signiﬁcant economic and social challenges to national healthcare systems. Machine Learning (ML) and advanced analytical tools have the potential to identify early disease biomarkers from small quantities of biological samples more quickly and cheaply than traditional methods.

Our approach integrates heterogeneous biological data to construct a predictive model for diagnosis using ML techniques. The model is then exploited to suggest microRNAs (miRNAs) as candidate biomarkers of disease onset and progression. Methodologically, we adopt a multi-view learning approach, which leverages various patient-related aspects (views), including miRNA expression values, sequences, and metadata. The novelty lies in the simultaneous use of these three views, complemented by a fourth that represents the interactions among miRNAs.

As a case study, we present the results from the analysis of Mild Cognitive Impairment (MCI) and Alzheimer’s (AD) patients using publicly available data.
We pre-processed and integrated four datasets from the Gene Expression Omnibus (GEO) repository, which were then fed to a multi-view Random Forest classiﬁer. Explainability techniques based on the importance of feature and permutation were adopted to identify candidate biomarkers. Finally, we conducted miRNA-target interaction (multiMIR) and functional pathway (DAVID, KEGG) analyses to interpret the results biologically.

High-scored predicted disease biomarkers will be validated by nanoﬂuidic qPCR analysis of miRNAs extracted from blood samples of AD and MCI patients under clinical observation. Preliminary results show signiﬁcant involvement of some potential miRNA biomarkers in neurological development and neurodegenerative processes. The ﬁnal goal is to diﬀerentiate between MCI and AD biomarkers.
