---
title: "Computational inference reveals cancer lineages and identifies early markers of tumor invasion"
author: "Marcel Tarbier"
authors:
  - name: Marcel Tarbier
    affiliations:
      - ref: u2
  - name: Almut S. Eisele
    affiliations:
      - ref: u1
  - name: Alexia A. Dormann
    affiliations:
      - ref: u1
  - name: Vicent Pelechano
    affiliations:
      - ref: u2
  - name: David M. Suter
    affiliations:
      - ref: u1
affiliations:
  - id: u1
    name: Ecole Polytechnique Fédérale de Lausanne, School of Life Sciences, Institute of Bioengineering; Lausanne, Switzerland
  - id: u2
    name: Science for Life Laboratory, Department of Microbiology, Tumor and Cell Biology, Karolinska Institute; Solna, Sweden
date: 2024-11-05T16:30
categories:
  - tools and software 2 session
  - tuesday

abstract: "Assigning single cell transcriptomes to cellular lineage trees by lineage tracing has transformed our understanding of differentiation during development, regeneration, and disease. However, lineage tracing is technically demanding and most scRNA-seq datasets are devoid of lineage information. Here we introduce Gene Expression Memory-based Lineage Inference (GEMLI), a computational tool allowing to robustly determine cell lineages solely from scRNA-seq datasets. GEMLI allows to study heritable gene expression, to discriminate symmetric and asymmetric cell fate decisions and to reconstruct individual multicellular structures from pooled scRNA-seq datasets. In human breast cancer biopsies, GEMLI revealed previously unknown gene expression changes at the onset of cancer invasiveness. The universal applicability of GEMLI allows studying the role of cell lineage trees in a wide range of physiological and pathological contexts. GEMLI is available as an R package on GitHub (https://github.com/UPSUTER/GEMLI)."
---